src/nodes/chunk_append/chunk_append.c

```diff
@@ -38,6 +38,27 @@ has_joins(FromExpr *jointree)
 	return list_length(jointree->fromlist) != 1 || !IsA(linitial(jointree->fromlist), RangeTblRef);
 }
 
+static void
```

Comment:
Could you add a comment on what the function does?

Reply:
added a comment